docs/get-started/java.md

```diff
@@ -252,7 +252,7 @@ Run your agent with the ADK web interface using the following Maven command:
 # Remember to load keys and settings: source .env OR env.bat
 mvn compile exec:java \
     -Dexec.mainClass="com.google.adk.web.AdkWebServer" \
-    -Dexec.args="--adk.agents.source-dir=target --server.port=8000"
+    -Dexec.args="--adk.agents.source-dir=. --server.port=8000"
```

Comment:
I wonder why `--adk.agents.source-dir=target` is working in my environment (where the compiled classes end up in `target/classes/com/example/agent`) but was not working for you and others in https://github.com/google/adk-java/issues/456? Do your compiled classes end up in a different location? Asking since we want to make sure we're matching common dev environments as accurately as possible.